python/bridgestan/model.py

```diff
@@ -32,22 +33,33 @@ class StanModel:
 
     def __init__(
         self,
-        model_lib: str,
-        model_data: Optional[str] = None,
+        model_lib: Union[str, PathLike],
+        data: Union[str, PathLike, None] = None,
```

Comment:
Is `None` better here than `""`?

Reply:
None ends up being passed as `nullptr` to C++, which then creates an `empty_var_context`:

https://github.com/roualdes/bridgestan/blob/41b66b7956817ecb411e7dccedeed4d6fd8347dc/src/model_rng.cpp#L65C1-L67C56

If you pass `""` instead, this string is then treated as an empty JSON, which creates a json parser/var_context and populates it with nothing. 

So they're semantically equivalent, but trigger different code paths